packages/@webex/plugin-meetings/src/breakouts/breakout.ts

```diff
@@ -60,33 +58,21 @@ const Breakout = WebexPlugin.extend({
    */
   async join() {
     const breakoutMoveId = uuid.v4();
+    const deviceUrl = this.webex?.internal?.device?.url;
```

Comment:
issue: none of these question marks are necessary. webex, internal, device should always be defined

Reply:
fixed